docs/guides/using-ronin-db/index.md

```diff
@@ -158,16 +158,22 @@ there are pending migrations. You will have to run `ronin-db migrate`.
 
 <div class="level">
   <div class="level-left">
-    <a class="button" href="../using-the-ronin-cli/">
-      Using the Ronin CLI &raquo;
+    <a class="button" href="../using-ronin-repos/">
+      &laquo; Using Ronin Repos
     </a>
   </div>
 
   <div class="level-item">
-    <a class="button" href="../index.html#guides">
+    <a class="button" href="/docs/#guides">
       &#x2303; Guides
     </a>
   </div>
+
+  <div class="level-right">
+    <a class="button" href="../ruby-quick-ref/">
+      Ruby Quick Ref &raquo;
+    </a>
+  </div>
```

Comment:
It don't make sense to link from the "Using the Ronin DB" guide to the "Ruby Quick Ref" guide, which is about scripting and writing Ruby.

Reply:
Okay i will change that!